deployment/helm/skaha/skaha-config/launch-carta.yaml

```diff
@@ -79,36 +79,26 @@ spec:
           protocol: TCP
           name: frontend-port
         volumeMounts:
+        - mountPath: "/skaha-system"
+          name: start-carta
```

Comment:
It looks like launch-carta.yaml is the only one that has the start script mount removed, and I'm thinking that was a mistake?

Reply:
Hello @brianmajor, We haven't removed the start-script mount. It seems the order of mounts has changed hence it might seem like its removed. 

```    
    volumeMounts:
        - mountPath: "/skaha-system"
          name: start-carta
        - mountPath: "/etc/passwd"
          name: etc-passwd
          subPath: passwd
        - mountPath: "/etc/group"
          name: etc-group
          subPath: group
        {{ template "skaha.session.commonVolumeMounts" . }}
      volumes:
      - name: etc-group
        emptyDir: { }
      - name: etc-passwd
        emptyDir: { }
      - name: start-carta
        configMap:
          name: launch-scripts
          defaultMode: 0777
      - name: init-users-groups
        configMap:
          name: init-users-groups-config
          defaultMode: 0777
      {{ template "skaha.session.commonVolumes" . }}
   ```